configurations/CSVToDynamo.hrf.json

```diff
@@ -0,0 +1,215 @@
+{
```

Comment:
Double check file name. I'm questioning the period between CSVToDynamo and hrf for file name consistency in the repo.